packages/react-native/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@aws-amplify/ui-react-native",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "tsc --project tsconfig.json",
+    "dev": "yarn build --watch",
+    "clean": "rimraf dist",
+    "lint": "tsc --noEmit && eslint src",
+    "test": "jest",
+    "test:coverage": "yarn test --coverage",
+    "test:watch": "yarn test --watch"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.17.10",
+    "@babel/core": "^7.17.10",
+    "@babel/preset-env": "^7.17.10",
+    "@testing-library/react-hooks": "^8.0.0",
+    "@types/react": "^17.0.2",
+    "@types/react-native": "^0.67.6",
+    "@types/react-test-renderer": "^18.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.20.0",
+    "@typescript-eslint/parser": "^5.20.0",
+    "babel-jest": "^28.0.3",
+    "eslint": "^8.14.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.1.4",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.4.0",
+    "metro-react-native-babel-preset": "^0.70.2",
+    "react": "^17.0.2",
+    "react-native": "^0.68.1",
+    "react-test-renderer": "^17.0.2",
+    "rimraf": "^3.0.2"
+  },
+  "dependencies": {
+    "@aws-amplify/ui": "3.7.0"
+  },
+  "peerDependencies": {
+    "react": ">= 16.8.0",
+    "react-native": ">= 0.63.4"
```

Comment:
Just curious, is there a reason we target this specific version and above?

Reply:
It is the lowest version of RN that supports the [Pressable API](https://reactnative.dev/docs/pressable)